server/test/ZCommentTest.js

```diff
@@ -16,6 +17,17 @@ const { expect } = chai;
 
 chai.use(chaiHttp);
 
+CommentModel.comments.push(
+  {
+    id: 1,
+    authorId: 2,
+    articleId: 1,
+    comment: 'Informative One!',
+    createdOn: '09/03/2019 12:04:59',
+    updatedOn: '09/03/2019 12:04:59',
+  },
+);
+
 const validToken = generateAuthToken(1);
 
 const invalidToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MiwiaWF0IjoxNTY3ODU2NTczfQ.WqwWVxxt9J8EN03toJM7K1QQBbTCJKe3lV-32axH';
```

Comment:
Move this token to the .env

Reply:
I am working on this ASAP

Reply:
I am done with this change https://github.com/hakoemmy/TeamWork/pull/32#discussion_r330899234